msfdb

```diff
@@ -1097,6 +988,20 @@ if $PROGRAM_NAME == __FILE__
 
   parse_args(ARGV)
 
+  update_db_port
+
+  if @connection_string
+    @db_driver = Standalone.new(options: @options, db_conf: @db_conf, connection_string: @connection_string)
+  elsif installed?('pg_ctl') && has_requirements(PgCtl.requirements)
+    @db_driver = PgCtl.new(db_path: @db, options: @options, localconf: @localconf, db_conf: @db_conf)
+  elsif installed?('pg_ctlcluster') && has_requirements(PgCtlcluster.requirements)
+    @db_driver = PgCtlcluster.new(db_path: @db, options: @options, localconf: @localconf, db_conf: @db_conf)
+  else
+    print_error('You must have pg_ctl or pgctl_cluster on your path, alternatively you can specify a database with `--connection-string`')
```

Comment:
This seems like a pretty low level message to show to the average user. Is it worth summarising that they most likely need to install postgres?

Reply:
what about "You need to have postgres install or specifcy a database with `--connection-string`" ?